src/GeniusYield/Types/Value.hs

```diff
@@ -598,9 +598,10 @@ parseAssetClassCore sep tkParser t = Atto.parseOnly parser (TE.encodeUtf8 t)
 
 -- | Token name is an arbitrary byte string up to 32 bytes long.
 --
--- TODO: it's unclear whether it's an arbitrary byte string
--- or UTF8 encoded text (which encoded byte form is 32 byte long at most).
+-- TODO: it's unclear whether it's an arbitrary byte string or UTF8 encoded text #32
+-- (which encoded byte form is 32 byte long at most).
 -- /We treat it as an arbitrary string/.
+-- (https://github.com/geniusyield/atlas/issues/32)
```

Comment:
could you move this above the "We treat it ..." line

Reply:
Fixed: b996d9d892a6d5587920d5cdb74e29217e3264b7